bpi.syndicate.inc

```diff
@@ -24,35 +32,120 @@ function bpi_syndicate_page_form($form, &$form_state) {
   if (bpi_service_is_configured()) {
     drupal_add_library('system', 'drupal.ajax');
 
-    $search_phrase = _bpi_current_search_phrase();
+    $query_form = array(
+      '#type' => 'container',
+      '#attributes' => array(
+        'class' => array(
+          'bpi-query',
+        ),
+      ),
+    );
+
+    $query_form['bpi_search'] = array(
+      '#type' => 'container',
+      '#attributes' => array(
+        'class' => array('bpi-search'),
+      ),
+    );
 
-    $form['bpi_search_query'] = array(
+    $query_form['bpi_search']['bpi_search_query'] = array(
       '#type' => 'textfield',
-      '#default_value' => $search_phrase,
+      '#default_value' => _bpi_get_current_search_phrase(),
+      '#description' => t('Search in body, teaser, title and category'),
+    );
+
+    $query_form['bpi_search']['bpi_search_actions'] = array(
+      '#type' => 'container',
+      '#attributes' => array(
+        'class' => array(
+          drupal_html_class('container-inline'),
+        ),
+      ),
     );
 
-    $form['bpi_search_submit'] = array(
+    $query_form['bpi_search']['bpi_search_actions']['bpi_search_submit'] = array(
       '#type' => 'submit',
       '#value' => t('Search'),
     );
 
-    $form['bpi_search_filters'] = array(
-      '#type' => 'item',
-      '#markup' => bpi_build_filters(),
-    );
+    if (_bpi_get_current_search_phrase()) {
```

Comment:
See comment about '_bpi_get_current_search_phrase()', which could change this to is !empty(_bpi_get_current_search_phrase()).